packages/kit/src/exports/public.d.ts

```diff
@@ -2058,7 +2059,16 @@ export type RemoteForm<Input extends RemoteFormInput | void, Output> = {
 	/** The number of pending submissions */
 	get pending(): number;
 	/** Access form fields using object notation */
-	fields: Input extends void ? never : RemoteFormFields<Input>;
+	fields: IsAny<Input> extends true
+		? RecursiveFormFields
+		: Input extends void
+			? {
+					/** Validation issues, if any */
+					issues(): RemoteFormIssue[] | undefined;
+					/** Validation issues belonging to this or any of the fields that belong to it, if any */
+					allIssues(): RemoteFormIssue[] | undefined;
+				}
+			: RemoteFormFields<Input>;
```

Comment:
I wonder if we can simplify this by just doing this:

```suggestion
	fields: RemoteFormFields<Input>;
```

Yes, you'll have `set` and `value` on a void form but you cannot really do anything with it anyway, so it's fine.